python/dglke/models/pytorch/loss.py

```diff
@@ -0,0 +1,94 @@
+from ..base_loss import *
+from .tensor_models import *
+import torch as th
+import torch.nn.functional as functional
+
+logsigmoid = functional.logsigmoid
+softplus  = functional.softplus
+sigmoid = functional.sigmoid
+
+class HingeLoss(BaseHingeLoss):
+    def __init__(self, args):
+        super(HingeLoss, self).__init__(args)
+
+    def __call__(self, score: th.Tensor, label):
+        loss = self.margin - label * score
+        loss[loss < 0] = 0
+        return loss
+
+class LogisticLoss(BaseLogisticLoss):
+    def __init__(self, args):
+        super(LogisticLoss, self).__init__(args)
+
+    def __call__(self, score: th.Tensor, label):
+        return softplus(-label * score)
+
+class BCELoss(BaseBCELoss):
+    def __init__(self, args):
+        super(BCELoss, self).__init__(args)
+
+    def __call__(self, score: th.Tensor, label):
+        return -(label * th.log(sigmoid(score)) + (1 - label) * th.log(1 - sigmoid(score)))
+
+class LogsigmoidLoss(BaseLogsigmoidLoss):
+    def __init__(self, args):
+        super(LogsigmoidLoss, self).__init__(args)
+
+    def __call__(self, score: th.Tensor, label):
+        return - logsigmoid(label * score)
+
+
+class LossGenerator(BaseLossGenerator):
+    def __init__(self, args):
```

Comment:
list all the parameters and give docs for each parameters. This func can be reused in KE Python API.